src/renderer/components/ft-subscribe-button/ft-subscribe-button.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <div class="buttons">
+      <ft-button
+        :label="subscribedText"
+        class="subscribeButton"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        @click="handleSubscription"
+      />
+      <ft-button
+        :aria-label="$t('Profile.Show Unubscribed Profiles')"
+        :title="$t('Profile.Show Unsubscribed Profiles')"
+        class="subIconButton plusButton"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        @click="showProfileList('subscribed')"
+      >
+        <font-awesome-icon
+          icon="plus"
+        />
+      </ft-button>
+      <ft-button
+        :aria-label="$t('Profile.Show Subscribed Profiles')"
+        :title="$t('Profile.Show Subscribed Profiles')"
+        class="subIconButton minusButton"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        @click="showProfileList('unsubscribed')"
+      >
+        <font-awesome-icon
+          icon="minus"
+        />
+      </ft-button>
+    </div>
+    <div
+      v-if="showProfiles==='subscribed'"
```

Comment:
I think it becomes much easier to read when there is spacing
```suggestion
      v-if="showProfiles === 'subscribed'"
```